images/manageiq-base/Dockerfile

```diff
@@ -61,36 +63,28 @@ RUN yum -y install centos-release-scl-rh && \
                    &&                      \
     yum clean all
 
-## GIT clone manageiq-appliance
 RUN mkdir -p ${APPLIANCE_ROOT} && \
-    curl -L https://github.com/ManageIQ/manageiq-appliance/tarball/${REF} | tar vxz -C ${APPLIANCE_ROOT} --strip 1
+    curl -L https://github.com/${APPLIANCE_ORG}/manageiq-appliance/tarball/${APPLIANCE_REF} | tar vxz -C ${APPLIANCE_ROOT} --strip 1
+
+RUN mkdir -p ${SUI_ROOT} && \
+    curl -L https://github.com/${SUI_ORG}/manageiq-ui-service/tarball/${SUI_REF} | tar vxz -C ${SUI_ROOT} --strip 1
```

Comment:
Is this needed in this layer?

Reply:
Maybe not?

I was trying to get all of the source in this image though for ease of development, but if it makes a big difference we could probably move this to the webservice image.

Reply:
It's actually quite small... 5.1MB currently for master.